tests/e2e/testcases/subnet/public/suite.go

```diff
@@ -249,4 +247,186 @@ var _ = ginkgo.Describe("[Public Subnet]", func() {
+
+		// obtain ledger1 addr
+		ledgerSimEndCh := utils.StartLedgerSim(0, 0, ledger1Seed, false)
+		ledger1Addr, err := utils.GetLedgerAddress(models.Local, 0)
+		gomega.Expect(err).Should(gomega.BeNil())
+		<-ledgerSimEndCh
+
+		// obtain ledger2 addr
+		ledgerSimEndCh = utils.StartLedgerSim(0, 0, ledger2Seed, false)
+		ledger2Addr, err := utils.GetLedgerAddress(models.Local, 0)
+		gomega.Expect(err).Should(gomega.BeNil())
+		<-ledgerSimEndCh
+
+		// obtain ledger3 addr
+		ledgerSimEndCh = utils.StartLedgerSim(0, 0, ledger3Seed, false)
+		ledger3Addr, err := utils.GetLedgerAddress(models.Local, 0)
+		gomega.Expect(err).Should(gomega.BeNil())
+		<-ledgerSimEndCh
+
+		// start the deploy process with ledger2
+		ledgerSimEndCh = utils.StartLedgerSim(3, 0, ledger2Seed, true)
+
+		// multisig deploy from unfunded ledger2 should not create any subnet/blockchain
+		gomega.Expect(err).Should(gomega.BeNil())
+		s := commands.SimulateMultisigMainnetDeploy(
+			subnetName,
+			[]string{ledger1Addr, ledger2Addr, ledger3Addr},
+			[]string{ledger2Addr, ledger3Addr},
+			txPath,
+			true,
+		)
+		toMatch := "(?s).+Ledger addresses:.+  " + ledger2Addr + ".+Error: insufficient funds.+"
+		matched, err := regexp.MatchString(toMatch, s)
+		gomega.Expect(err).Should(gomega.BeNil())
+		gomega.Expect(matched).Should(gomega.Equal(true))
+
+		// let's fund the ledger
+		err = utils.FundLedgerAddress(genesis.MainnetParams.TxFeeConfig.CreateSubnetTxFee + genesis.MainnetParams.TxFeeConfig.CreateBlockchainTxFee)
+
+		// multisig deploy from funded ledger2 should create the subnet but not deploy the blockchain,
+		// instead signing the tx and creating the tx file to wait for ledger3's signature
+		gomega.Expect(err).Should(gomega.BeNil())
+		s = commands.SimulateMultisigMainnetDeploy(
+			subnetName,
+			[]string{ledger1Addr, ledger2Addr, ledger3Addr},
+			[]string{ledger2Addr, ledger3Addr},
+			txPath,
+			false,
+		)
+		toMatch = "(?s).+Ledger addresses:.+  " + ledger2Addr + ".+Subnet has been created with ID.+" +
+			"1 of 2 required Blockchain Creation signatures have been signed\\. Saving tx to disk to enable remaining signing\\..+" +
+			"Addresses remaining to sign the tx\\s+" + ledger3Addr + ".+"
+		matched, err = regexp.MatchString(toMatch, s)
+		gomega.Expect(err).Should(gomega.BeNil())
+		gomega.Expect(matched).Should(gomega.Equal(true))
+
+		// wait for end of ledger2 simulation
+		<-ledgerSimEndCh
+
+		// try to commit before signature is complete (no funded wallet needed for commit)
+		s = commands.TransactionCommit(
+			subnetName,
+			txPath,
+			true,
+		)
+		toMatch = "(?s).*1 of 2 required signatures have been signed\\..+" +
+			"Addresses remaining to sign the tx\\s+" + ledger3Addr +
+			".+Error: tx is not fully signed.+"
+		matched, err = regexp.MatchString(toMatch, s)
+		gomega.Expect(err).Should(gomega.BeNil())
+		gomega.Expect(matched).Should(gomega.Equal(true))
+
+		// try to sign using unauthorized ledger1
```

Comment:
This should probably be a separate test case, but we can tackle this in a future issue